data/json/recipes/recipe_deconstruction.json

```diff
@@ -81,81 +81,113 @@
+    "difficulty" : 0,
+    "time" : 500,
+    "reversible" : true,
+    "autolearn" : false,
     "qualities" : [ {"id":"SCREW","level":1,"amount":1} ],
-    "components": [
+    "components" : [
       [ [ "scrap", 1 ] ],
       [ [ "plastic_chunk", 2 ] ],
       [ [ "rag", 2 ] ],
       [ [ "RAM", 1 ] ],
       [ [ "e_scrap", 2 ] ]
     ]
-  }, {
+  },{
     "type" : "recipe",
-    "result": "bowl_pewter",
-    "id_suffix": "uncraft",
-    "category": "CC_NONCRAFT",
-    "subcategory": "CSC_NONCRAFT",
-    "difficulty": 0,
-    "time": 500,
-    "reversible": true,
-    "autolearn": false,
-    "qualities":[
-      {"id":"HAMMER","level":1,"amount":1}
+    "result" : "bowl_pewter",
+    "id_suffix" : "uncraft",
+    "category" : "CC_NONCRAFT",
+    "subcategory" : "CSC_NONCRAFT",
+    "difficulty" : 0,
+    "time" : 500,
+    "reversible" : true,
+    "autolearn" : false,
+    "qualities" : [ {"id":"HAMMER","level":1,"amount":1} ],
+    "components" : [
+      [ [ "tin", 100 ] ]
+    ]
+  },{
+    "type" : "recipe",
+    "result" : "cu_pipe",
+    "id_suffix" : "uncraft",
+    "category" : "CC_NONCRAFT",
+    "subcategory" : "CSC_NONCRAFT",
+    "difficulty" : 0,
+    "time" : 1200,
+    "reversible" : true,
+    "autolearn" : false,
+    "qualities" : [ {"id":"HAMMER","level":1,"amount":1} ],
+    "tools": [
+      [ [ "toolset", 10 ] ],
+      [ [ "fire",-1 ] ]
     ],
-    "components": [ [ [ "tin", 100 ] ] ]
+  "components" : [
+      [ [ "copper", 200 ] ]
+    ]
   },{
-  "type" : "recipe",
-  "result": "cu_pipe",
-  "id_suffix": "uncraft",
-  "category": "CC_NONCRAFT",
-  "subcategory": "CSC_NONCRAFT",
-  "difficulty": 0,
-  "time": 1200,
-  "reversible": true,
-  "autolearn": false,
-  "qualities":[ {"id":"HAMMER","level":1,"amount":1} ],
-  "tools": [
-    [
-      [ "toolset", 10 ],
-      [ "fire", -1 ]
+    "type" : "recipe",
+    "result" : "silver",
+    "id_suffix" : "uncraft",
+    "category" : "CC_NONCRAFT",
+    "subcategory" : "CSC_NONCRAFT",
+    "difficulty" : 0,
+    "time" : 500,
+    "reversible" : true,
+    "autolearn" : false,
+    "qualities" : [ {"id":"HAMMER","level":1,"amount":1} ],
+    "components" : [
+      [ [ "silver_small", 800 ] ]
     ]
-  ],
-  "components": [ [ [ "copper", 200 ] ] ]
-},{
+  },{
     "type" : "recipe",
-    "result": "silver",
-    "id_suffix": "uncraft",
-    "category": "CC_NONCRAFT",
-    "subcategory": "CSC_NONCRAFT",
-    "difficulty": 0,
-    "time": 500,
-    "reversible": true,
-    "autolearn": false,
-    "qualities":[ {"id":"HAMMER","level":1,"amount":1} ],
-    "components": [ [ [ "silver_small", 800 ] ] ]
+    "result" : "small_relic",
+    "id_suffix" : "uncraft",
+    "category" : "CC_NONCRAFT",
+    "subcategory" : "CSC_NONCRAFT",
+    "difficulty" : 0,
+    "time" : 500,
+    "reversible" : true,
+    "autolearn" : false,
+    "qualities" : [ {"id":"HAMMER","level":1,"amount":1} ],
+    "components" : [
+      [ [ "silver_small", 100 ] ]
```

Comment:
Entries like this could be condensed to a single line without harming readability